diff --git a/node_modules/google-translate-api/index.js b/node_modules/google-translate-api/index.js
index d142733..f3a2613 100644
--- a/node_modules/google-translate-api/index.js
+++ b/node_modules/google-translate-api/index.js
@@ -30,9 +30,9 @@ function translate(text, opts) {
     opts.to = languages.getCode(opts.to);
 
     return token.get(text).then(function (token) {
-        var url = 'https://translate.google.com/translate_a/single';
+        var url = 'https://translate.google.cn/translate_a/single';
         var data = {
-            client: 't',
+            client: 'gtx',
             sl: opts.from,
             tl: opts.to,
             hl: opts.to,
